src/components/interface/capmgr/memmgr.h

```diff
@@ -10,4 +11,23 @@ int memmgr_shared_page_alloc(vaddr_t *pgaddr);
 int memmgr_shared_page_allocn(int num_pages, vaddr_t *pgaddr);
 int memmgr_shared_page_map(int id, vaddr_t *pgaddr);
 
+/* This magic number is double the tls size defined in RK */
+#define TLS_AREA_SIZE 32
+#define TLS_NUM_PAGES (round_up_to_page(TLS_AREA_SIZE * MAX_NUM_THREADS) / PAGE_SIZE)
+#define TLS_BASE_ADDR 0x70000000
```

Comment:
None of these should be here. It is perhaps implementation specific, so either have it `capmgr` implementation if it isn't used outside that or have it somewhere like `cos_types.h` if it's required outside of the implementation.